engine/server/sv_game.c

```diff
@@ -143,18 +142,20 @@ void SV_SetModel( edict_t *ent, const char *name )
 	ent->v.model = MAKE_STRING( sv.model_precache[i] );
 	ent->v.modelindex = i;
 
-	mod_type = Mod_GetType( ent->v.modelindex );
+	model = Mod_Handle( ent->v.modelindex );
+	VectorCopy( model->mins, ent->v.mins );
+	VectorCopy( model->maxs, ent->v.maxs );
```

Comment:
This would lead to useless work. SV_SetMinMaxSize will set values to entity's mins and maxs.